tap_helpscout/transform.py

```diff
@@ -77,11 +77,46 @@ def transform_conversations(this_json, path=None):
     return this_json
 
 
+def get_rating_value(rating_id):
+    return {"1": "Great", "2": "Okay", "3": "Not Good"}.get(rating_id, "")
```

Comment:
Best practice has been to return the raw API value (such as we do for tap-intercom), so I think we should get Product input before we move forward with inferring the meaning of an integer field in order to create a field that isn't native to the endpoint.

I can understand that this information might be more usable for an end-user, but it creates additional maintenance burden on our end if Helpscout were to change the interpretation of ratings in the future (which could otherwise just be publicized or already-known to users of the third-party service)

Reply:
Product confirmed this should be excluded.